app-drawer-layout/app-drawer-layout.html

```diff
@@ -156,7 +119,6 @@
           type: Boolean,
           readOnly: true,
           notify: true,
-          reflectToAttribute: true,
```

Comment:
I think we actually want to keep this one so one can style stuffs based on the existence of `narrow` attribute.  For example, in the publishing template here: https://github.com/PolymerLabs/app-layout/blob/drawer/templates/publishing/src/blog-app.html#L100